src/user/user.model.ts

```diff
@@ -7,12 +7,36 @@ const userSchema = new mongoose.Schema<IUser>({
     required: true,
     unique: true,
     lowercase: true,
+    IsEmail: true,
```

Comment:
the mongoose package, from which we define the schema doesn't have "IsEmail" and "IsString" validators. you should remove them here and also everywhere else.  "IsEmail" and "IsString" are used in dto classes to validate incoming data using the class-validator package.